llvm/include/llvm/Target/TargetSelectionDAG.td

```diff
@@ -959,6 +959,7 @@ class PatFrags<dag ops, list<dag> frags, code pred = [{}],
   list<dag> Fragments = frags;
   code PredicateCode = pred;
   code GISelPredicateCode = [{}];
+  code GISelRegPredicateCode = [{}];
```

Comment:
Move this down into PatLeaf? Or should we have a new subclass of PatLeaf, similar to ImmLeaf?